modules/features/manatime_feature_timesheet_reminders/manatime_feature_timesheet_reminders.features.field_instance.inc

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * @file
+ * manatime_feature_timesheet_reminders.features.field_instance.inc
+ */
+
+/**
+ * Implements hook_field_default_field_instances().
+ */
+function manatime_feature_timesheet_reminders_field_default_field_instances() {
+  $field_instances = array();
+
+  // Exported field_instance: 'message-timesheet_reminder-field_user'.
+  $field_instances['message-timesheet_reminder-field_user'] = array(
```

Comment:
We don't need this field. Delete it please. (And update the message text so that it uses the message author instead of this field)